README.rst

```diff
@@ -0,0 +1,105 @@
+Flask-SQLAlchemy
```

Comment:
Use the same style readme as Flask please. I'm trying to be consistent across Pallets projects.